src/theory/theory_engine.h

```diff
@@ -403,6 +404,10 @@ class TheoryEngine : protected EnvObj
       CDHashMap<NodeTheoryPair, NodeTheoryPair, NodeTheoryPairHashFunction>
           PropagationMap;
 
+
+  std::unique_ptr<cvc5::theory::Splitter> d_splitter;
+  int d_numPartitions;
+  std::list<Node> d_assertedPartitions;
```

Comment:
Data members [go last](https://github.com/cvc5/cvc5/wiki/Code-Style-Guidelines#declaration-order). It would also be good to add a short comment to those.